tensorboard/plugins/graph/keras_util_test.py

```diff
@@ -285,11 +293,11 @@ def test_keras_model_to_graph_def_lstm_model(self):
       attr {
         key: "keras_class"
         value {
-          s: "LSTM"
+          s: "%s"
```

Comment:
Letting you know: instead of using LSTM which for some reason had changed its name, I decided to use `SimpleRNN` (or I can use any other layer technically; most AFAICT didn't change in name unlike the LSTM).

While moving this to proto, I found myself confused a little. 

Is proto construction more readable and is it easier to reason whether our utility is doing the right thing or do you think pbtxt is much more readable at glance (when it is small in size, of course)? If string interpolation makes pbtxt less appealing, it should be "addressed" by using SimpleRNN. I realized I hate the `lstm_class_name` I previously took and it is unavoidable even with proto construction.